app/models/dependant_income_model.rb

```diff
@@ -34,7 +34,7 @@ def income_too_high
 
   class << self
     def dependant_monthly_upper_limit
-      361.70
+      FeatureFlags.enabled?(:thresholds_2025, without_session_data: true) ? 367.87 : 361.70
```

Comment:
@willc-work I thought the new limit was meant to be £367.87 and the olf limit was meant to be £361.70.

So should this not be
 FeatureFlags.enabled?(:thresholds_2025, without_session_data: true) ? 367.87 : 361.70